src/thunderbots/software/ai/navigator/placeholder_navigator/placeholder_navigator.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include "ai/intent/intent.h"
+#include "ai/intent/visitor/intent_visitor.h"
+#include "ai/navigator/navigator.h"
+#include "ai/primitive/primitive.h"
+
+/**
+ * This PlaceholderNavigator is a simple navigator that will convert the given Intents
+ * into their respective Primitives, without doing any intelligent navigation. It is meant
+ * to provide an example of how to implement a navigator and act as a starting point for
+ * future navigators to build off of.
+ */
+class PlaceholderNavigator : public Navigator, public IntentVisitor
+{
+   public:
+    explicit PlaceholderNavigator() = default;
+
+    std::vector<std::unique_ptr<Primitive>> getAssignedPrimitives(
```

Comment:
Can we add a javadoc comment for this function?

Reply:
Don't think it needs one since it's overriding the function from the `Navigator` superclass, which has the comment there?